free-1.md

```diff
@@ -45,6 +69,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+To be completely honest, if this thing exists, I would have heard about it. I follow the dota2 reddit, so if there was a community accessible tester for the game, either as a DSL or a GUI, I'd know. Its possible that one exists for developers of the game, but it would be internal, so of no use to the general population.
 
```

Comment:
A little more work to discover if there's something out there would be good. I googled "Dota 2 simulation" and found the [Dota 2 Calculator](http://devilesk.com/dota2/apps/hero-calculator/), but I'm not sure how related it is.